### DISCUSSION TITLE
Excluding files from build

### DISCUSSION THREAD

**Discussion by Kimmottaja** [neutralinojs/neutralinojs#720] [Q&A] [answered]
Is it possible to exclude files in the resources folder ending up in res.neu file?

**Comment by shalithasuranga** [comment, marked as answer]
Hey.. Thanks for the idea. Right now the CLI will include all files. But we can add a new feature. Could you let us know your directory structure and which files you need to exclude if possible? So we can check further :)

**Reply by Kimmottaja** [reply]
In my projects I often use sass to handle css stylesheets, and somehow really dislike the thought of the .scss files being left in the build, even though they don't take up any significant space.

Well, the obvious solution was to move the .scss files outside the resources folder and just set their compile paths to point to it.
(And this is what i've done)

But as I do nearly all work in the resources folder I thought it might be nicer, if the .scss files could also 'live' in there and just be left out when I build the application.

My directory structure looks a bit like this:

```
|-- dist
|-- resources // everything in here seems to end up in the res.neu -file
|   |-- js
|   |   |-- neutralino.js
|   |   |-- main.js
|   |
|   |-- styles
|   |   |-- sass // This folder is what I'd like to leave out of the res.neu -file
|   |   |   |-- style.scss
|   |   |
|   |   |-- compiled_style.css // Because it's contents have already been compiled to here
|   |
|   |-- index.html
|
|-- neutralino.config.json
|-- package.json
```

**Reply by shalithasuranga** [reply]
Hey.. Thanks for posting more details. Yeah, like you mentioned moving source files to another directory is a solution for most scenarios, when we use React (https://github.com/codezri/litepy), etc too. Maybe we can add a pattern to exclude some files from the build process. Eg: `*.scss`. I will post an update here if we introduce a feature like that :)

Issue created: https://github.com/neutralinojs/neutralinojs-cli/issues/80
